src/database/schema/ChatSchema.ts

```diff
@@ -175,6 +174,16 @@ class ChatSchema implements BaseDbSchema {
     return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
   }
 
+  static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+    const selectQuery = `SELECT A.id
+      FROM ${ChatSchema.getTableName()} A
+      WHERE A.id = ?;`;
+
+    const [{rows}] = await db.executeSql(selectQuery, [id]);
+    if (rows.length === 0) return Promise.resolve(null);
+    return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new `isExists` method has been added to check if a chat exists in the database by its ID. The implementation looks correct, but as mentioned earlier, it should return a boolean value instead of a `ChatSchema` object.
```diff
- static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+ static async isExists(db: SQLiteDatabase, id: string): Promise<boolean> {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->